src/node_jsvmapi.h

```diff
@@ -22,6 +22,14 @@
 #include "node_macros.h"
 #include "node_jsvmapi_types.h"
 
+#define NAPI_METHOD(name)                                                      \
+  void name(napi_env env, napi_callback_info info)
+#define NAPI_GETTER(name) NAPI_METHOD(name)
+#define NAPI_SETTER(name) NAPI_METHOD(name)
+
+#define NAPI_MODULE_INIT(name)                                                 \
+  void name(napi_env env, napi_value exports, napi_value module)
+
```

Comment:
IIRC we were gonna drop `NAPI_MODULE_INIT` and `NAPI_METHOD` because the signatures are not supposed to change.